jerry-core/ecma/builtin-objects/ecma-builtin-helpers.c

```diff
@@ -481,6 +480,96 @@ ecma_builtin_helper_string_index_normalize (ecma_number_t index, /**< index */
   return norm_index;
 } /* ecma_builtin_helper_string_index_normalize */
 
+/**
+ * Helper function for finding lastindex of a search string
+ *
+ * This function clamps the given index to the [0, length] range.
+ * If the index is negative, 0 value is used.
+ * If the index is greater than the length of the string, the normalized index will be the length of the string.
+ * NaN is mapped to zero or length depending on the nan_to_zero parameter.
+ *
+ * See also:
+ *          ECMA-262 v6, 21.1.3.9
+ *
+ * Used by:
+ *         - The ecma_builtin_helper_string_prototype_object_index_of helper routine.
+ *         - The ecma_builtin_string_prototype_object_replace_match helper routine.
+ *
+ * @return uint32_t - whether there is a match for the search string
+ */
+static ecma_value_t
```

Comment:
Based on the comment above this should return "uint32_t" and not "ecma_value_t" which one is the correct one?